docs/src/pages/demos/tables/DenseTable.tsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { createStyles, Theme, withStyles, WithStyles } from '@material-ui/core/styles';
+import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
 import Table from '@material-ui/core/Table';
 import TableBody from '@material-ui/core/TableBody';
 import TableCell from '@material-ui/core/TableCell';
 import TableHead from '@material-ui/core/TableHead';
 import TableRow from '@material-ui/core/TableRow';
 import Paper from '@material-ui/core/Paper';
 
-const styles = (theme: Theme) =>
+const useStyles = makeStyles((theme: Theme) =>
   createStyles({
```

Comment:
we should no longer need `createStyles`

Reply:
Should have kept it all in one PR, would have made this easier for all of us.

See https://github.com/mui-org/material-ui/pull/15390#discussion_r277050903